### ISSUE TITLE
Explain & * usage in creating variable groups example of preprocessor episode

### ISSUE THREAD

**Issue by sverhoeven** [ESMValGroup/ESMValTool_Tutorial#135]
The example uses `&` and `*`, if as reader of the tutorial I would see that the first time in this example. It would help if those chars are explained to me. 

I found https://docs.esmvaltool.org/projects/esmvalcore/en/latest/recipe/overview.html#re-using-parameters-from-one-script-to-another and https://learnxinyminutes.com/docs/yaml/ which give a bit of explanation that could be linked to in a callout.

**Comment by Peter9192** [comment]
I agree that this warrants extra (dedicated) explanation. That could be done in 'special topics', as suggested in #142.
